src/Microsoft.DotNet.Build.Tasks/PackageFiles/Symbols.targets

```diff
@@ -403,4 +448,24 @@
                        Condition="'@(CatalogFile)'!=''" />
   </Target>
 
+  <!--
+    Generates Windows PDBs from Portable PDBs.
+    
+    [In]
+    @(PortableFileToConvert)
+      * ItemSpec: The DLL file associated with a Portable PDB to convert.
+      * PdbPath: The path to the Portable PDB file to convert.
+      * TargetPath: The output path for the generated Windows PDB. Full filename.
+    @(ConversionOptions) [optional]
+      * ItemSpec: An entry in the Microsoft.DiaSymReader.Tools.PdbConversionOptions flags enum to
+        use for all conversions performed.
+  -->
+  <Target Name="CreateWindowsPdbsFromPortablePdbs">
+    <Error Text="BuildTools does not support Portable PDB conversion to Windows PDB in .NET Core. Run msbuild using the desktop framework."
```

Comment:
MSBuildRuntimeType is `core` for Linux, I'm guessing?  Does this break there or does `SubmitSymbolsRequest` never get called?  I'd be fine with skip and warn instead of error out if we're unsure about effects on client repos.